docs/user_guide/mitigation.rst

```diff
@@ -15,13 +15,13 @@ Fairlearn contains the following algorithms for mitigating unfairness:
       - binary classification
       - regression
       - supported fairness definitions
-   *  - :code:`fairlearn.` :code:`reductions.` :code:`ExponentiatedGradient`
+   *  - :class:`fairlearn. reductions. ExponentiatedGradient <fairlearn.reductions.ExponentiatedGradient>`
```

Comment:
why the spaces after the dot?

Reply:
It's because the table columns have only very limited space. If I don't have that the other columns become unreadable. This splits the link over three lines (which we've had like that for over a year).

Reply:
Do we really want the whole `fairlearn.reductions.ExponentiatedGradient` path? We could have only the `ExponentiatedGradient` displayed and the link would tell the users where the object is:

```suggestion
   *  - :class:`~fairlearn.reductions.ExponentiatedGradient`
```

Or   

```
- :class:~`fairlearn.reductions.ExponentiatedGradient`
```

Reply:
Going with the latter.